ui-app/client/src/App/appStyleProperties.ts

```diff
@@ -422,6 +432,14 @@ export const styleProperties: Array<StylePropertyDefinition> = [
 		noPrefix: true,
 	},
 
+	{
+		groupName: 'Color',
+		displayName: 'Color Sixteen',
+		name: 'colorSixteen',
+		defaultValue: '#1F1F1F',
+		noPrefix: true,
+	},
+
```

Comment:
Why do we need another color? We already have #333333 it is not much of a difference from that one.

Reply:
we already have the color this is not required